src/sentry/static/sentry/app/components/modals/suggestProjectModal.tsx

```diff
@@ -0,0 +1,260 @@
+              <ListItem>
+                {tct(
+                  '[solve:Solve] issues quickly with full context: contextualized stack traces, events that lead to the error, client, hardware information, and the very commit that introduced the error.',
+                  {
+                    solve: <strong />,
+                  }
+                )}
+              </ListItem>
+              <ListItem>
+                {tct(
+                  '[learn:Learn] and analyze event data to reduce regressions and ultimately improve user adoption and engagement.',
+                  {
+                    learn: <strong />,
+                  }
+                )}
+              </ListItem>
+            </StyledList>
+
+            <SmallP>{t('And guess what? Setup takes less than five minutes.')}</SmallP>
+          </ModalContainer>
+        </Body>
+        <Footer>
+          <Access organization={organization} access={['project:write']}>
+            {({hasAccess}) => (
+              <ButtonBar gap={1}>
+                <Button
+                  priority={hasAccess ? 'default' : 'primary'}
+                  onClick={this.handleAskTeammate}
+                >
+                  {t('Tell a Teammate')}
+                </Button>
+                {hasAccess && (
+                  <Button
+                    href={newProjectLink}
+                    onClick={this.handleGetStartedClick}
+                    priority="primary"
+                  >
+                    {t('Get Started')}
+                  </Button>
+                )}
+              </ButtonBar>
+            )}
+          </Access>
+        </Footer>
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {Header} = this.props;
+    const {askTeammate} = this.state;
+    const header = askTeammate ? t('Tell a Teammate') : t('Try Sentry for Mobile');
+    return (
+      <React.Fragment>
+        <Header>
+          <PatternHeader />
+          <Title>{header}</Title>
+        </Header>
+        {this.state.askTeammate ? this.renderAskTeammate() : this.renderMain()}
+      </React.Fragment>
+    );
+  }
+}
+
+const ModalContainer = styled('div')`
+  display: grid;
+  grid-gap: ${space(3)};
+
+  code {
+    word-break: break-word;
+  }
+`;
+
+const Title = styled('h3')`
+  margin-top: ${space(2)};
+  margin-bottom: ${space(3)};
+`;
+
+const SmallP = styled('p')`
+  margin: 0;
+`;
+
+const PatternHeader = styled('div')`
+  margin: -${space(4)} -${space(4)} 0 -${space(4)};
+  border-radius: 7px 7px 0 0;
+  background-image: url(${bgPattern});
+  background-size: 475px;
+  background-color: black;
+  background-repeat: no-repeat;
+  overflow: hidden;
+  background-position: center bottom;
+  height: 156px;
+`;
+
+const StyledList = styled(List)`
+  li {
+    padding-left: ${space(3)};
+    :before {
+      top: 7px;
+    }
+  }
+`;
+
+const BackWrapper = styled('div')`
+  width: 100%;
+  margin-right: ${space(1)};
+`;
+
+const StyledBackButton = styled(Button)`
```

Comment:
How about setting `BackWrapper` to `display: flex; justify-content: flex-end` and you don't need this styled component?

Reply:
The problem is the div around the primary CTA here

![Screen Shot 2021-02-17 at 11 49 18 AM](https://user-images.githubusercontent.com/8533851/108259666-38237880-7116-11eb-80f9-8a1d619fcce1.png)